mflowgen/tile_array/construct.py

```diff
@@ -236,8 +241,9 @@ def construct():
   g.connect_by_name( Tile_PE,      drc          )
   g.connect_by_name( Tile_PE,      lvs          )
 
-  g.connect_by_name( rtl,         dc        )
-  g.connect_by_name( constraints, dc        )
+  g.connect_by_name( rtl,            dc        )
+  g.connect_by_name( constraints,    dc        )
+  g.connect_by_name( dc_postcompile, dc        )
```

Comment:
Am I reading this correctly? `dc_postcompile` happens *before* `dc`?

Reply:
dc_postcompile contains scripts that the dc node runs, similar to the constraints file